### DISCUSSION TITLE
konga conenction

### DISCUSSION THREAD

**Discussion by ahmettortumlu** [v1bh0r/konga-gke#3] [Q&A]
At this architecure how can i connect and manage to konga, to connect from browser I will need to [Public-IP]:1337 ? Do you have any idea about that?

**Comment by v1bh0r** [comment]
There's a load balancer defined in [https://github.com/v1bh0r/konga-gke/blob/master/manifests/konga-deployment.yaml](https://github.com/v1bh0r/konga-gke/blob/master/manifests/konga-deployment.yaml)
It exposes konga on port 80. The load balancer service will display a public IP which you can use to access the application.

**Reply by v1bh0r** [reply]
On a different note, for production deployment, I would recommend using an ingress service which creates a load balancer behind the scenes.

[https://cloud.google.com/kubernetes-engine/docs/tutorials/configuring-domain-name-static-ip](https://cloud.google.com/kubernetes-engine/docs/tutorials/configuring-domain-name-static-ip)

**Reply by ahmettortumlu** [reply]
Thank you @v1bh0r i understand the consept and will check them.

**Reply by ahmettortumlu** [reply]
I should note that i couldn't connect to konga, just used different VPC, I couldn't connect to konga and used kong's public ips. I guess I should do additional steps to reach public IP addresses.
